docs/install.md

```diff
@@ -7,7 +7,7 @@ control plane.
 !!! note "TL;DR"
 
     ```bash
-    kubectl apply -f https://docs.k0smotron.io/{{{ extra.k0smotron_version }}}/install.yaml
+    kubectl create -f https://docs.k0smotron.io/{{{ extra.k0smotron_version }}}/install.yaml
```

Comment:
This is fine for the makefile and fine for new installations, but raises the question on how to update.

I guess for now we can merge this but we should add a note after 1.0 saying that in order to upgrade do kubectl patch instead. Anyway I don't think this is a high priority right now... so LGTM

Reply:
Yes… `kubectl apply --server-side=true` does the trick as well, should we use it instead?

Reply:
I think I'd prefer using `--server-side=true`.